l10n-pocket/en/pocket/add.ftl

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://dev.tekcopteg.com/add/
+
+pocket-add-how-to-save = How to Save
+pocket-add-save-to-pocket = Save to { -brand-name-pocket } from { -brand-name-firefox }
+pocket-add-click-the-pocket-button = Click the { -brand-name-pocket} button in your { -brand-name-firefox } toolbar to save articles, videos, pages, and more.
+pocket-add-learn-how = Learn how
+pocket-add-dont-see-the-pocket = Don’t see the { -brand-name-pocket } buttton?
+
+pocket-add-view-from-any = View from any Device
+pocket-add-different-devices = Different devices displaying { -brand-name-pocket } articles
+# this is a title for a list of different app stores where you can download pocket
+pocket-add-get-pocket-for = Get { -brand-name-pocket } for:
+
+pocket-add-apple-app-store = { -brand-name-apple-app-store }
+pocket-add-mac-app-store = { -brand-name-mac-short } app store
+pocket-add-google-play-store = { -brand-name-google-play } store
+pocket-add-amazon-app-store = { -brand-name-amazon } app store
+pocket-add-windows = { -brand-name-windows }
+pocket-add-kobo-e-reader = { -brand-name-kobo } e-reader
+pocket-add-windows-mobile = { -brand-name-windows-mobile }
+pocket-add-blackberry = { -brand-name-blackberry }
+pocket-add-more-apps = More Apps
```

Comment:
**Question:** Shouldn't these strings [be under `brands.ftl`](https://github.com/mozilla/bedrock/blob/main/l10n-pocket/en/brands.ftl)? Since they're going to be used site-wide most likely. I'm raising this question because of this comment @alexgibson left on my PR: https://github.com/mozilla/bedrock/pull/11638#discussion_r873662365.